app.js

```diff
@@ -32,4 +37,24 @@ io.on('connection', function(socket) {
   socket.on('disconnect', function() {
     disconnectMessages.forEach(doEmit);
   });
+  socket.on('connect_error', function(error) {
+      debug('ERROR: ' + JSON.stringify(error));
+  });
 });
+
+function report() {
+    var end = process.hrtime();
+    if (stats.last_update) {
+        var delta_t_sec = (((end[0] - stats.last_update[0]) * 1000000000) +
+            (end[1] - stats.last_update[1])) / 1000000000;
```

Comment:
That's fine.  I added another parens around the entire division, to make that part a little clearer.